src/sdam/state/server.rs

```diff
@@ -29,31 +33,24 @@ impl Server {
         }
     }
 
-    pub(crate) fn new(
+    pub(crate) fn create(
```

Comment:
This seems fine to me, although I think it would be useful to add a doc comment indicating that the monitoring tasks are started as part of this method.

More generally, since we're starting to get in the habit of starting larger numbers of tasks, I think we should also make it a habit to note that in the doc comments of a method when doing that.

Reply:
So this method (currently) doesn't start any monitoring tasks, though it probably should. The only reason it doesn't is that weird lock shenanigans I mentioned in my other response to one of your comments. If we go ahead with that change I'll be sure to update this docstring to indicate the monitor gets started.

Reply:
Oh, I missed the fact that monitors need to be explicitly started now rather than starting when creating. Never mind!